index.html

```diff
@@ -5,13 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
-    <link href="./main_style.css" type="text/css" rel="stylesheet" />
-    <script src="https://kit.fontawesome.com/6a5b34451f.js" crossorigin="anonymous"></script>
+    <link href="styles/main_styles.css" type="text/css" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/6a5b34451f.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
   </head>
   <body>
     <header>
-      <div class="logo">My logo</div>
-
+      <div class="logo"><a href="#">My logo</a></div>
+      <span class="fa fa-bars" id="menu"></span>
```

Comment:
- [ ] According to the project requirement and previous review, the hamburger menu should have a `nav` tag.